### DISCUSSION TITLE
Should a new TM4E release be prepared?

### DISCUSSION THREAD

**Discussion by rubenporras** [eclipse/tm4e#657] [General]
I have noticed that the latest release is already half a year old. Would it be worth having a newer release?

**Comment by sebthom** [comment]
I was waiting for the IP approval for the language pack update which we got a few days ago. so yes a new release would be worth it. I am going to prepare it once I got feedback from @mickaelistria regarding the issue #654 he reported yesterday.

**Reply by sebthom** [reply]
0.9.0 has been released.

**Reply by akurtakov** [reply]
And submitted to simrel https://github.com/eclipse-simrel/simrel.build/pull/170
